models/articles.model.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../db/connection');
+
+exports.fetchArticles = (articleId) => {
+    let whereId = '';
+    const values = [];
+    if(articleId){
+        whereId = ` WHERE article_id = $1`;
+        values.push(articleId);
+    }
+    return db.query(`
+        SELECT * FROM articles
+        ${whereId}
+    `, values)
+    .then((data) => {
```

Comment:
This has been over engineered, and will cause you issues in further tickets when it comes to adding queries.
A seperate model for fetching all articles and for fetching by article ID is needed.

Reply:
fixed